packages/edit-site/src/components/global-styles/font-library-modal/style.scss

```diff
@@ -1,4 +1,11 @@
 .font-library-modal {
+	&.is-full-screen {
+
+		@include break-medium() {
+			width: 65vw;
```

Comment:
Thanks for the review! 

> Can we build this into the modal component itself?

Does this mean that the width is controlled by JavaScript rather than a stylesheet? If so, I can change the approach to something like this, for example:

```javascript
function FontLibraryModal() {
	const isLargeViewport = useViewportMatch( 'medium' );

	return (
		<Modal
			style={ { width: isLargeViewport ? '65vw' : undefined } }
		></Modal>
	);
}
```

Or do you mean adding width constraints to the base modal component itself?

```scss
.components-modal__frame {
	&.is-full-screen {
		@include break-medium() {
			width: 65vw;
		}
	}
}
```

Reply:
I meant the latter, that ideally we have an excellent _default_ behavior for the modal component, and we provide overrides on a per-component basis only as an exception. 

I've heard someone mention an idea to build in "small", "medium", and "large" props into the modal, so that there's some variety in responsive behaviors. But all that boils down to is to have fewer local overrides, ideally. What do you think?

Reply:
Well, that might make sense. Currently, there are only two width variations for the modal component: the default auto-width or the full-width with the `isFullScreen` prop opted in.

However, the Font Manager has only recently been implemented, so as the UI and UX improve, the width may change, or this override style itself may become unnecessary, or may be displayed in full screen.

Considering this, I think there is no problem in fixing the bugs and pushing forward with PR for now.

Reply:
Hey all, we've been starting work on this very feature last week — a [PR](https://github.com/WordPress/gutenberg/pull/54471) is up, although I haven't had the time to take a look at it just yet.

Reply:
@t-hamano , I wonder if you'd like to hold on with the work on this PR, in case https://github.com/WordPress/gutenberg/pull/54471 lands and we could use that instead.

Reply:
@ciampo Thank you for teaching me that. I think #54471 is exactly what this PR needs, so I'm happy to hold this PR .